README.md

```diff
@@ -1,5 +1,13 @@
 # Getting Started
 
+## Officially supported exchanges
+
+* [Binance](partners/binance/)
+* [Coinbase Pro](officially_supported/coinbase_pro.md)
```

Comment:
I used 3 level to catégorie exchanges. To me a partner exchange is also officially supported.

Reply:
Agree but to me it's not clear when coming to this page to see which exchanges are supported and not seeing "binance" in supported exchanges (it means people have to scroll and read more in the page, which most wont do). I think it's better to list all supported exchanges in this section even if it lists them twice (or make a ccxt like table maybe ? with a mark when partners

Reply:
Sounds good to me